docs/topics/proc_configure-netty-logging-vertx.adoc

```diff
@@ -0,0 +1,32 @@
+
+[#_configure-netty-logging-vertx]
+= Configuring Netty Logging
+
+link:http://netty.io/wiki/user-guide-for-4.x.html[_Netty_^] is a library used by VertX to manage asynchronous network communication in applications.
```

Comment:
Why is the link italicised? The convention is to use product and project names without any special mark-up, IIRC.

Reply:
true, let me de-mark-up it:)